lib/LabeledTextItem/LabeledTextItem.less

```diff
@@ -1,7 +1,7 @@
 /* Labeled Text Item */
 .moon-labeledtextitem {
 	min-width: 336px;
-	height: 192px;
+	height: auto;
```

Comment:
Lets remove the lines for `height` and `margin: 0px;`.
* `height: auto` isn't needed (like @ryanjduffy said in Jira) since height is already inherently "auto". One less rule to parse, yay!
* margin has no effect, as it's naturally got no margin. Don't need a rule to reiterate it.